starter-code/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
You're making a mistake adding the settings into starter-code and ignoring it above, in the root folder. Can you see it?